openfecwebapp/templates/macros/breadcrumbs.html

```diff
@@ -1,21 +1,19 @@
 {% macro breadcrumbs(title, links) %}
-<header class="page-header page-header--primary">
-  <ul class="breadcrumbs">
-    <li class="breadcrumbs__item"><a href="{{ cms_url }}" class="breadcrumbs__link" rel="Home">Home</a></li>
-    {% for link in links %}
-    <li class="breadcrumbs__item">
-      {% if link[0] != '' %}
-        <span class="breadcrumbs__separator">&rsaquo;</span>
-        <a class="breadcrumbs__link" href="{{ link[0] }}">{{ link[1] }}</a>
-      {% else %}
-        <span>{{ link[1] }}</span>
-      {% endif %}
-    </li>
-    {% endfor %}
-    <li class="breadcrumbs__item breadcrumbs__item--current">
+<ul class="breadcrumbs">
```

Comment:
Ah. I see why you changed this from generating the entire page header, but this requires changing the way this macro gets used on all the other templates that use it. (I shouldn't have called it `breadcrumbs` since it does all the other stuff.)

Do you want to revert this or update the rest of the templates?